src/utils/kukai.utils.ts

```diff
@@ -55,12 +44,15 @@ const bumpIV = (salt: string, bumps: number) => {
 
 export const decryptSeedPhrase = async (json: string, pwd: string) => {
   const walletData = JSON.parse(json);
+  if (walletData.walletType === 4 && (walletData.version === 1 || walletData.version === 2)) {
```

Comment:
are you sure that only `walletType: 4` should not be decrypted? mb remove that condition and check only `various`